src/_pytest/python_api.py

```diff
@@ -105,6 +140,66 @@ def __repr__(self) -> str:
         list_scalars = _recursive_list_map(self._approx_scalar, self.expected.tolist())
         return f"approx({list_scalars!r})"
 
+    def repr_compare(self, other_side: "ndarray") -> List[str]:
+        import itertools
+        import math
+
+        def _get_value_from_nested_list(
```

Comment:
```suggestion
        def get_value_from_nested_list(
```

It is a local method, no need for the `_`. 👍